realm: Replace occurences of uri with url. (#25141)
zulip/zulip: api_docs/changelog.md

## Changes in Zulip 7.0

**Feature level 177**:

> * [`POST /register`](/api/register-queue):

Reviewer: This should mention both of the endpoints that changed.

Author: Yes, I have added [`[POST /server_settings](/api/get-server-settings)`](https://github.com/zulip/zulip/blob/407b4b4172580c8ded9d181514d888089f43eb9d/api_docs/changelog.md?plain=1#L26)

## Changes in Zulip 8.0

**Feature level 202**:

* [`POST /register`](/api/register-queue), [`POST /server_settings`](/api/get-server-settings),
  `PATCH /realm`:
  Rename `realm_uri` to `realm_url` and keep the deprecated name `realm_uri` for
  backwards-compatibility.

> * Mobile push notifications now include the `realm_url` field as an alias
>   of `realm_uri` in one of common fields for all notification payloads.

Reviewer: ```suggestion
* Mobile push notifications now include a `realm_url` field as an alias
  of `realm_uri` in all notification payloads.
```
